### DISCUSSION TITLE
Is there a performance gain with GTSAM_BUILD_WITH_MARCH_NATIVE ?

### DISCUSSION THREAD

**Discussion by TobiasLangner-TomTom** [borglab/gtsam#1760] [Q&A]
I'm optimizing a simple SLAM problem with pose (Pose3) and observation (Point3) variables. The factors are various priors and between factors. The problem sizes are rather large with thousands of poses and observations.
TBB is not enabled because multi-threading is managed outside of GTSAM. 

According to [INSTALL.md](https://github.com/borglab/gtsam/blob/develop/INSTALL.md#performance) building with the option `GTSAM_BUILD_WITH_MARCH_NATIVE` should bring a performance gain of 25-30%. However, I did not observe any improvement for my use case. 

I would be grateful if anyone could share their experiences with GTSAM_BUILD_WITH_MARCH_NATIVE .
Does anyone know how the 25-30% were measured? 
Which parts of GTSAM would benefit the most from GTSAM_BUILD_WITH_MARCH_NATIVE?

**Comment by ProfFan** [comment]
You definitely need to measure on your own platform, please refer to the Eigen documentation for details.
